sale_force_cancel_cft/security/security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<openerp>
+    <data>
+        <record id="group_force_cancel_so" model="res.groups">
```

Comment:
Please add following, to ensure that after install this, admin (root) will have this group by default,

            <field name="users" eval="[(4, ref('base.user_root'))]"/>